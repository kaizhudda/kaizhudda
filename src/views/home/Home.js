import React from 'react';
import HeroBanner from './heroBanner';
import Profile from './profile';
import Summary from './summary';

import './Home.scss';

const Home = () => {
  return (
    <div className="Home">
      <HeroBanner />
      <Summary />
      <Profile />
    </div>
  );
}

export default Home;
