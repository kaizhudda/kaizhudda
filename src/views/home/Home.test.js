import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

let wrapped = shallow(<Home />); 

describe('Home', () => {
  it('should render the Home Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should render HeroBanner Components', () => {
    const heroBanner = 'HeroBanner';
    expect(wrapped.find(heroBanner).exists()).toBe(true);
  });

  it('should render HeroBanner Components', () => {
    const heroBanner = 'HeroBanner';
    expect(wrapped.find(heroBanner).exists()).toBe(true);
  });

  it('should render Summary Components', () => {
    const summary = 'Connect(Summary)';
    expect(wrapped.find(summary).exists()).toBe(true);
  });

  it('should render Profile Components', () => {
    const profile = 'Profile';
    expect(wrapped.find(profile).exists()).toBe(true);
  });

  it('should render Skills Components', () => {
    const skills = 'Skills';
    expect(wrapped.find(skills).exists()).toBe(true);
  });

  it('should render Contact Components', () => {
    const contacts = 'Contact';
    expect(wrapped.find(contacts).exists()).toBe(true);
  });

});
