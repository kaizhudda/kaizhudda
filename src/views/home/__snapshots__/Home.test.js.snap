// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the Home Component correctly 1`] = `
<div
  className="Home"
>
  <HeroBanner />
  <Connect(Summary) />
  <Profile />
  <Skills />
  <Contact />
</div>
`;
